content/reference-transaction-format.md

```diff
@@ -724,6 +725,125 @@ In the above example with a ¥95/$15 offer and a ¥5/$2 offer, the situation is
+  "PublicKey": "32D2471DB72B27E3310F355BB33E339BF26F8392D5A93D3BC0FC3B566612DA0F0A"
+}
+```
+
+<!--{# TODO: replace the above example with one where the channel, pubkey, signature, and balance match #}-->
+
+| Field       | JSON Type | [Internal Type][] | Description                    |
+|:------------|:----------|:------------------|:-------------------------------|
+| `Channel`   | String    | Hash256           | The unique ID of the channel, as a 64-character hexadecimal string. |
+| `Balance`   | String    | Amount            | _(Optional)_ Total amount of XRP, in drops, delivered by this channel after processing this claim. Required in order to deliver XRP. Must be more than the total amount delivered by the channel so far, but not greater than the `Amount` of the signed claim. Must be provided except when closing the channel. |
+| `Amount`    | String    | Amount            | _(Optional)_ The amount of XRP, in drops, authorized by the `Signature`. This must match the amount in the signed message.  |
+| `Signature` | String    | VariableLength    | _(Optional)_ The signature of this claim, as hexadecimal. The signed message contains the channel ID and the amount of the claim. Required unless the sender of the transaction is the source address of the channel. |
+| `PublicKey` | String    | PubKey            | _(Optional)_ The public key used for the signature, as hexadecimal. This must match the `PublicKey` stored in the ledger for the channel. Required unless the sender of the transaction is the source address of the channel and the `Signature` field is omitted. (The transaction includes the PubKey so that `rippled` can check the validity of the signature before trying to apply the transaction to the ledger.) |
+
+### PaymentChannelClaim Flags
+
+Transactions of the PaymentChannelClaim type support additional values in the [`Flags` field](#flags), as follows:
+
+| Flag Name | Hex Value  | Decimal Value | Description                         |
+|:----------|:-----------|:--------------|:------------------------------------|
+| `tfRenew` | 0x00010000 | 65536         | Clear the channel's `Expiration` time. (`Expiration` is different from the channel's immutable `CancelAfter` time.) Only the source address of the payment channel can use this flag. |
+| `tfClose` | 0x00020000 | 131072        | Request to close the channel. Only the channel source and destination addresses can use this flag. This flag closes the channel immediately if it has no more XRP allocated to it after processing the current claim, or if the destination address uses it. If the source address uses this flag when the channel still holds XRP, this schedules the channel to close after `SettleDelay` seconds have passed. (Specifically, this sets the sets the `Expiration` of the channel to the close time of the previous ledger plus the channel's `SettleDelay` time, unless the channel already has a sooner `Expiration` time.) |
```

Comment:
This might be covered elsewhere, but what happens to the XRP if the channel still holds XRP when it closes?

Ah, I see:

> After the `Expiration` time, the channel closes automatically and returns any unspent XRP to the sender. 

Might be worth adding that explanation here as well.